src/test/javascript/spec/component/exam/exam-update.component.spec.ts

```diff
@@ -45,7 +46,8 @@ describe('Exam Update Component', function () {
     const exam = new Exam();
     exam.id = 1;
 
-    const course = { id: 1 } as Course;
+    const course = new Course();
+    course.id = 1;
```

Comment:
Do we really want to extract this like that? I think it adds unnecessary lines and is less compact.

Reply:
This has the benefit of the Course having default values for most properties, I assume the cast would lead to undefined errors in one place or the other.